src/orm/generateORM.ts

```diff
@@ -59,19 +66,32 @@ export class OrmGenerator {
         }
         hasExclamationToken = false;
       }
+      let decorators;
+      if (field.columnType !== undefined) {
+        const name = SpecialORMColumnType[field.columnType];
+        this.importManager.addImport('typeorm', [name]);
+        decorators = [
+          {
+            name: name,
```

Comment:
field.columnType ?? SpecialORMColumnType.COLUMN

instead of condition

Reply:
fixed